packages/core/nuxt-theme-module/theme/composables/useUiNotification/index.ts

```diff
@@ -2,12 +2,12 @@ import { computed, reactive } from '@vue/composition-api';
 
 interface UiNotification {
```

Comment:
We talked about differentiating the `useUiNotification` type and `send` params because `id` is assigned automatically